### DISCUSSION TITLE
How to use Dropout

### DISCUSSION THREAD

**Discussion by kkmcgg** [NSCC-COGS/Aestheta#62] [Guides]
https://www.machinecurve.com/index.php/2019/12/18/how-to-use-dropout-with-keras/

**_Dropout_**, as its called, is a pretty slick way to help avoid overfitting your model. Essentially what you do is instruct the model to _turn off_ a certain percentage of neurons in a given layer randomly for each training epoch. This has the rather magical effect of forcing the network to find _other_ pathways to descend toward the minimum loss. It's kind of like putting little obstacles in its way - forcing the neural net to _think on its feet_!
